src/utils/actions.js

```diff
@@ -19,11 +20,11 @@ const actions = [
     group: 'bookmark',
     getActionCall:
       ({ app, bookmark }) =>
-        async () => {
-          const bookMarks = await app.getBookmarkList();
-          const findBm = bookMarks.find((bm) => bm.qData.title === bookmark);
-          bookmark && (await app.applyBookmark((findBm && findBm.qInfo && findBm.qInfo.qId) || bookmark));
-        },
+      async () => {
```

Comment:
Should there not be an indent here?

Reply:
I [had a try](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACSKAZ3wHM4YA1AQwBsBXOAGQEsS8BePaogTyjB4AFNQwYANHgBudRkUlsAItXgBKTgD48wADpQ8eWhWmy4RAILZs1XpxMMzAOiIZaLGEIDkAbk+rvegYsAGbCSipw6rr6BobGACYRAKKY2GZELNB2nj6BsTIOFlY2jsE4SdRgABZCQolqmtp5sXj1cCkYaUQZWQDUXAAGAHL0ALZeACTAbQC+fgBkPp7zAwExBjP+zQTQ7IRScLhw8crwRHbUAO7U7txijnAF9BF1yanpmVDObmBwQgAMkgAtABWVRbdb2OSWay2Lj7Q7wE4RIjOVzuLy+CEbPJpGD0bD6J5mGElUaiWrE9QcDTbEajABGh0cbCG1CGQl6VLwAH5tHgAI4AFXQMCQULgeBmeHFwEFAEkiPTDiwwOKYNhGJIBfSmdhxbrDkJuTM8hCZms9IR2JUYJ9zlwANp5aItYnizyiVy8ABCEAgAGtydgA55xNsNdRiLQVJ8PQB5BkAKzgYBgjnMac+PvoMBg0AzYlovv9QeoIbD21I2Ag9AwHoZpeDofDkPIMEzdugAGE6LQkNsDEI5V7JI3A82pdTaZCDDx+IIhNOmrPYtb8OOAwBZcsBh3ca63L2Odt+ie71gkJdrFotdd4YIsKDxH2jOybnch1GP5+1BmjZd-0cAVTmoRw7RgIxOA4LhN2bbFbwMODdzweZ5mEK4bnwY8vWLM8yxDWofxfN80IfJ8SOA+UoDKVD0OI18qJoiAqPidQAB92LwZCQ3BG9bxmVsWjSAV6BYNJ4mojBc3FR1PB40MAF0hKlFTXXyUwG0qFsI2saNY2gBNk1TdNO2zXN8y+LcIAOH1tOubB4krNsazrD0iDgIws2gZyWnbMyez7AdV2HO4MCnTRB24PgBGEZd1MQzCj3uBltOvKLBO2ESxIkqSZLwR1lLyTKYgSiUPTKbAHKclSDEjfSuygIyUzTDNvKgHM8wLayDgAMRwarfNiata3rPBPA8rzGqGgx-Pa3taH7KLQq9CKaSi+dYqXRoypaJLsPuSrqvS1cSuEuBRPE448rFAqipiM6VzdTTxrAIxy3MRaZrweqiBjRrmpMtrGs6yzHG7d7sE+2hvpGtzxsmkzPlhigAqgBalpCkcxEkIgIGCGAmAgMAAzWmdEpixd4qiudDwOjBHDeuAPsWoQ8YJomSYQlpHoMbKrskqBpNuuT2cJ4mlJUx6yvdV7IehzqhSqNhvt+-643GxMWtM9rQYLCHmahxbFeVohYdcsaJs8pGfNqvA5sajHgsQlaccFABlGAIjZUY4EkR9PPiXH8fFkmyY2ym4p2mm8BCYQA9oNinsQtddnwBP4i1i46bCk8KF6lhA6EDPtU973qF97mU-28jA61xn5eN3MlbYNmQ85gMq55jK7f53KhfyuSM7DBH24lzx7o2NTtllzwmfLAvA9VvS-oMprNeM1q0b1r4DfLd3rfa6jF8T83Rvcw-prth3Pid5bsYkD2vfgH2-drxPw9XTaqej1cglCYuhdE5RBjgYe8Gcs5cBrsedsJ94iAMDqXZ+cBX5d1vDXCByYG6GxOinKUPcsoXRytdAeItPDD0nqpF0M8XpWymtDZeUZV4Aw3trYG5kupfAPvQxax8gE1SrBbC+U1kbX1RvNIK98wpIPLr7f2-Dg4cwlp-CmC4o40mTohOOCDgGaJTuA-hkCDxYVzrA-hOig5P1kZEfieCMGGKwYjNM0M25KK5rYgSBDIR9xIcLWS5D+EjwmmPEmE8pbT0hLPJxVBTBmztmrNegMt66wsgWbhJkaCFD4Uva+QiEaX1EVWcRjtJFY2kVYl+Fc34lwlIo0OpNNiRS-pHbaGjdqxG0RnOiEoQF-1TsQdODikzZxMTA-O5iakCjLpUyuHjEIGMDtRWiUCc5jJgHAxQZgwDYBYBgRqFi0F3jThKS8+AVkmPbJkxgpyRBu2JPIxZzFgJCmoKQVET43r0HiGYSYbQ-CHNiPYuujjL5XJ+cSU5-s6AeTqR3AFppTq9yIQLG6-jh6SE8MSIJYsO5hOKhE56DhhEmR3DAaoT5SBgribpJh6tDKsKBtvVJXDL6kvJVASlsSz7wzoTbKAKMOwSMWs7W8rtH5TOQa-B5idJD3LwDi5RjT1rNLUa0rQ7T-7x34d07kGr+nsEwcM85yUGZmKLpM6ZKCqkAtpiYw1zhL7GlMFC2gML5UhM7nM2ICLEK8zwD4wWfiCoBJyeNLFGKFWhModLGhRL8k8PgISWMBxGENQ1p4LWjKUmcMcOk5xtBE1QGTWYblltomFLbMU2+pSXYPxkTM6pCj3VuIadOCOqrqZ9M6fw3peCwHHMNSMk1ed1kTKbRK6xNrjG3HtdEz6hbi2uPqfCrx51Lr9yDUPQJEaPV4oegSjScbeVphSJ875AjIQJJYRmzeOsQbMtzZfU9DBz2luJe1AVaM75lNHBUq1cj36WMja2ppqitqdrwd2wOva+0LMTkY6B9wzWJwsfW-9NjQHTsGcCpMDqprPq+ccJdcKvU4kRYQ9dvjB4htPjulte6p7UMibQ6JAAFCA3QWAMiMKm5h6bM3JPvTmvNMB2Oce43AKlb7418s-UKzGtbykTobdKoDHqVHoJaRBlOUHdF6v7QMwDCHVlIfGea8dlrUGkcBTnWdl8xMZAk8RiWK7yPeORRu6j6LR70ejQegws98ykFIEYETvG6XrxvWwplOahQQGC0YOBImK1+Tyceq+RTBUlOFVI39yn0Oqdlc65t9SNN7S07-SDACulkV1ZhuDmdkxDvpiOuBqG-1Wcw0C+DWCgshbgCJp1DhYUuesz6gSSLKOBq89usNphsW7r80xwljAPS0Algw+JK8ItJLvRwsGHdobJd2NJ9LKXhpVsCjln9YgyuAoq20mOiGGbrZJi4tBfqA2oruuE5bh7VvjVewGOB4XEkMsE-tgsHcQe5PPjJj9YisvVuu4pvLlmqmqbu3OB76qY66aTvpnYhnB3Gpa8h+BFrJXWus7amdQzHBA9wSncbPNJvEOm2Qihv3SqxoB54egUAgebZpWm+lUWs1CbBgAVUFxtxax3iCnfLbbTLX6a2ioflj6KHbKvVxMwzAXQvWYfbZyi0hskls8+Y0ew3EsYci742LgTe3oA70cDLoX9uXJw7OyrytSOrsKY10p9HAGulKvJppnXj2u3VZ7Xo+ZA6hnNdMWZlDlPJ006w0ZrBtuSZM99au2IX3zfBq5-iv7AWWMUBoDs6gEnQfXud+w13D6D4xLrw32HPKZCd6MNS-3auUfB9-b3lg9ejBFYcFr7+6jcex+EGPifkpaumBg3gjUthCcGfYEviTxnRmmY7+PiTPpeCvydX3jDfS9o5z30YeuHkYCex2RysFQ3GBTulGAFQ1RhCRATxTgAHogC8AoAIAgQIAMBMMWdPE3M112dvs5J784AgksVLcDAZZaF4gIAAAlTyCAagC9FoK9fjW9FvDqB9RQPAggog07NIdbOgxHIfIPFoMVSQDAcsO0OgGfHHQAnffASUUnXOHA-Axg+BQEPAAAQikM4NwHH1oBdQ8gBW0QAMJ2e0cBwPdmoAOALzgN9VN08zITkO4JhgwL0SAIACpLC8BrC7D7CHDHCnDnDrDBwbCUhUxcxJRzBcwIByRGoicvY0xzgnwF1aA8ACRXUfoIA8AIAGQvYnxuBfD-DPg3DY5mJsAUisgox4g8B0BPD4AfoqhJRqBki155VL5jh0iijJROgoDERbBOCoBPJ0i0jjhbhRgIBvlHA3C0jzBcCABxd2NI1aIEPAfA4IQ4BAX4aIggAkNIWAMKdI4onZJEB8GsN8CidARwJMIgEYsogIsYmAYo2OeATY3I-GGopI-MLI-QfMQI20K49wOAUYc4UQFgfYm4teMTGAUCPAMY8cd6IkUwRwPAeUUIDULUAgNIFEbgbiJsFCS444uANIsAeYhAfAZXYgbgZ8Co58c4ZE-1MwegAtCleE88EMWOXIngHE9I4WNIuQqpRNWYwk6JKo0or4xqHo2cSwoAvQB3HbcHF3SgnNDwtE+AHwzk87KvI9fI8UuASUvwteOTbLVg2Idg64pUxqSQDkrUz4H4v4iPdtcDXXLRABXU246QmCCI58OAH8Y4dfFOTffg9BA4z4OwaYrouAKXXA+UbsPwrAZo2AEQN06AKdAQ04l4pZGI4QyYslGoAYRwRwIA94oAqQAARiAOeNeKAKmAtLXhmAGFUHAmKKgFqC6EDOUMaArN2DgB2LxjLL4kwwjN1OjJTzjOqCEETOTNTIzJTNDOIFzOAGzOjMcC6FrGwF+HlHiELOLORLLKEBrOIAAOVT7VvCXI8nrOgCXGbLwHDLiEELQAKLgFYxUCqDsG7JTIwBYDTMzPzMaiICHNbKeVIDEhnJTPakfLlK8J5AAA0gQxTcxPggQ4sAwEAOA8zfDRzvzGoAB9fMMCqAQsrPTVEMqU6AA0lQagR0tc+8J8YWH0IHfcDBCgTsy83su8gcx8qYEcp5ccgkKc987jCWIgIs3cloEshAcsswSslcrQDcus3Y7c8Edi2ITihc5ikmIgbTNcxCIwfAbM-cQqFClOSqYQeS2OOwf4bwTSgAHm4iIscEkr3AZwQFIGOJ0pYF6F6BwtkpaG0WMtREcsdBYEUk0LYFcBsFfmgi4E8BujNlsrsvsrOP3EcqMqIpcrcsqlGBUrXIZBhIDFiuZ1Eu7hvz7TxAJH0EUqSo2H3NQvwtzEIpYtMo5WOLwC0H+ECtg2OWaMuFOCELALgEuDwHqr0LsvvH-ADFYxrHCi4G3xTgFG6qgNlBStvAFGjJGrSrsvGviHFAGHvM+FgrzPuBYBnKWuAAWugBmHWtqvqpHSFBYF9iXELLtiCrGqFF4AwDgAbARIrFOrOqoSmr7QFC3AoGoE2WCEmoeqdPcCMDmoGIQEOAiGpIHPJIIlJiyCmF2oiHAggHlHdnjBfwpWOoGHuu+u+SIG2V2QBlGrwU8ABuaOsDWPuLqKkFWslC9mwHbE1MtMuHcHPIUlmOprRKsAxIqJESyBIAiG+m+oMFgu2WZiRDP3cigCBFtGAoZGZR5t5v5phKRH6n1HGk2v5TRoetlsFuOHjHXgGChqar2vzHhsRo1GRtUBOtxtvFgtWrmuVvWs9O+R9L9IDOgAxNuQZlWtNvWuVu2t1rqphvbAOqOtNtRvNsYyeoMNGotFGo6tGFJmEK9HNsZjlrgHwmbCEE6qGowFUATvnL-Fus7kaAUhHSYBsFrA8BErDtvHEtqBjF4FLuXBrtLqYjKFYjyps0PwZiIB0LgGdyIDatkpgpPLPIvKmAHtPOOJmHmBotISKqksdH+DcpjCZFoFhqJkuEOF7A8mOoguADtu9N9P9NGEDJdvNuPFWtGtNsnuHOnoivTIXvr08hXogDXuwA3r+FNu3s6uQtEtgLwR-uANAIHquOVt3JIvjKEFHrPLnNLO4pcFrOXAEq3MbK-wIF-3PPAaqpaBALAIgKgJgJjj-s+w8yozIWVoY1iEmJUAJDgF6hjFIA9HMG7CFHlHjCGFgp9ClyFCFBYdgvMA4fjC3HMCYZYfdiGhjWtz5zSGCC6CqEUH4ArlVEoELkuAHxIO2zB3Fwh1bxzQmOkdkaLVGAUaUZUeGjS3iDkYMbAEUaauMdmku3RnVzYLlA8mXLZDJtIHKKNJVRNJjyq2EBgEurgEuOcY8lcZYHca5Mkd0fMcMesZ8vGmCAFw-Qwez2CatTcbXjHLtKif0ZieUb7rI3DvcymyQMoQMAofxDSBodeQ9FwKSF6lqfdgAAlYLFAABNIYcwLcP02CygeUJIAAdXdlgvjEUHMAGNgtY3jFYyl1Y1EdbEUktGaMwBwHwHvBvl2FOTsCEDYCSCLQkyTg0XFt2FKBYALSNBEGXCkLAnKaoaqdIDwE4ljiIF2eX3gWucFtudoabKgD0HQCwFwAfESYCOqFTADHdiqCfrRleC9lUxg3vEeKgU-JOd-BEHamXHqDAiOagAuquriaxccCpH4gysJG4DTG6VtEyeKdIXJdahL2FhZFtLQHjGCAOStK4CBHTLWARV+eWYBe+WCFKILVJYfLWBAHEBACgIfOQFAHLBrEuFPIWKIGQBADoGuF4CVfFfiu0goG0N9lYGaOQAFddT9hAC1ZJh1c4LAApWQEhJNZeKZHiHPWLo5WeHIAVv8LtA5WVd1LFZACqBgFGFoH6eVjOEtYGwQAyDtDJv8eVbAG6F9afA8lwG6teXJENehRNd2LQHdgpSMAAEV6AIB4B03jXxW5Ck3lXUol7fXOhQj+nVrjjkAAAOQEEAOojyfp6wDAZVzoMwQ4A4X10SItk8nqpVlAHgIEZo44Y4X1nxFN0gNNpAI1mFEADyAxm1zUE1jIDlfNwt4tpdjN8Vr2Bket+IRtpAAAJiPesFOYpQPsXdQCIBBF9foA8heQZDHeXZNakEYGom+VgHdixr2XMGfE9l4CMBLY8hmBmCAA). Prettier decides not.